newIDE/app/src/UI/Menu/ElementWithMenu.js

```diff
@@ -5,45 +5,61 @@ import ContextMenu from './ContextMenu';
 
 type Props = {|
   element: React$Element<any>,
-  buildMenuTemplate: () => Array<any>,
+  onClick?: () => void,
+  buildMenuTemplate?: () => Array<any>,
+  openMenuWithSecondaryClick?: boolean,
 |};
 
 type State = {||};
 
 /**
  * Wrap an element and display a menu when `onClick` prop is called on the element.
  */
+
 export default class ElementWithMenu extends React.Component<Props, State> {
   _contextMenu: ?ContextMenu;
   _wrappedElement: ?any;
 
-  open = (event: any) => {
-    const { _contextMenu } = this;
-    if (!_contextMenu) return;
+  open = (element: any) => {
+    if (!this.props.buildMenuTemplate) {
```

Comment:
Do we need the changes in this function? Presumably this component is always used with a `buildMenuTemplate` prop? Not having it is an error no?.